gitlint/__init__.py

```diff
@@ -163,6 +163,22 @@ def get_vcs_root():
     return (None, None)
 
 
+def processFile(filename, modified_files, commit, force, gitlint_config):
+    vcs, _ = get_vcs_root()
+
+    if force:
+        modified_lines = None
+    else:
+        modified_lines = vcs.modified_lines(filename,
+                                            modified_files[filename],
```

Comment:
Just pass the value corresponding to the filename

Reply:
Done.